python/GangaTest/__init__.py

```diff
@@ -5,6 +5,8 @@ def getEnvironment(c = None):
     return {}
 
 def loadPlugins( config = None ):
+    import Lib.GListApp
```

Comment:
After some problems on getting the GListApp to import properly I though I may as well just put it in GangaTest with the other testing based classes. It made sense and made sure it was imported properly (with for example, the TestSubmitter)